.github/workflows/ci.yaml

```diff
@@ -143,55 +211,38 @@ jobs:
 
   format:
     runs-on: ROCM-Ubuntu
```

Comment:
Do we really need to use ROCM-Ubuntu for this?

Reply:
Not sure, what's a better alternative?

Reply:
I looked around at other PRs and couldn't find one where the `format` job took much time at all.  So your comment is reasonable and I changed it to use the free 2 core container instead.   If `format` ever becomes the long pole it can be revisited